### PR TITLE
Fix MongoId validator

### PR DESCRIPTION
**Pull request by wajdijurry**
Hello!

* Type: bug fix | code quality
* Link to issue:
https://github.com/phalcon/incubator/issues/914

**In raising this pull request, I confirm the following (please check boxes):**

- [*] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/incubator/blob/master/CONTRIBUTING.md)?
- [*] I have checked that another pull request for this purpose does not exist.
- [*] I wrote some tests for this PR.

Small description of change:
Update MongoId validator with new MongoDB driver.
Mongo extension is deprecated, and the validator should be updated.

Thanks

### CODE REVIEW

File: tests/unit/Validation/Validator/MongoIdTest.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php
+/**
```
Comment (CameronHall) [inline tests/unit/Validation/Validator/MongoIdTest.php]: Can you please remove this or create a proper docblock?

Comment (wajdijurry) [reply tests/unit/Validation/Validator/MongoIdTest.php]: Removed

File: Library/Phalcon/Validation/Validator/MongoId.php [hunk 2]
```diff
@@ -14,16 +14,16 @@
...
   +------------------------------------------------------------------------+
   | Authors: Anton Kornilov <kachit@yandex.ru>                             |
+  | Maintainer: Wajdi Jurry <jurrywajdi@yahoo.com>
   +------------------------------------------------------------------------+
 */
 
 namespace Phalcon\Validation\Validator;
 
-use MongoId as Id;
+use \MongoDB\BSON\ObjectId;
```
Comment (Jeckerson) [inline Library/Phalcon/Validation/Validator/MongoId.php]:
```suggestion
use MongoDB\BSON\ObjectId;
```

File: Library/Phalcon/Validation/Validator/MongoId.php [hunk 3]
```diff
@@ -36,30 +36,22 @@ class MongoId extends Validator
...
-                    $attribute,
-                    'MongoId'
-                )
-            );
+        if ($value instanceof ObjectId || preg_match('/^[a-f\d]{24}$/i', $value)) {
+            return true;
         }
 
-        return $result;
+        $message = ($this->hasOption('message')) ? $this->getOption('message') : 'MongoId is not valid';
```
Comment (Jeckerson) [inline Library/Phalcon/Validation/Validator/MongoId.php]:
```suggestion
        $message = $this->hasOption('message') ? $this->getOption('message') : 'MongoId is not valid';
```

**Comment by wajdijurry** [comment]
@Jeckerson all suggested changes are committed.
Thanks

**Comment by wajdijurry** [comment]
Hello,
I see that the pull request still not merged, could you please tell me if there is something missed from my side ?

**Pull request merged**